examples/multiplayer-netfox-state-machine/scripts/player-spawner.gd

```diff
@@ -0,0 +1,57 @@
+extends Node
```

Comment:
Note to self: We could move this to examples/shared later on, so it's not duplicated, and easier for users to copy and paste into their project for a quick start. 
Not necessary in this PR though, we can do it later.